cluster/examples/kubernetes/ceph/object-multisite-pull-realm.md

```diff
@@ -0,0 +1,54 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-multisite.yaml
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
+kind: CephObjectRealm
+metadata:
+  name: realm-a
+  namespace: new-rook-ceph-namespace
+spec:
+  pull:
+    endpoint: http://10.103.133.16:80
```

Comment:
How about a big comment that makes it obvious that this needs to be changed in their cluster?